### PR TITLE
Fastdmm test

### LINKED ISSUE #13719: Repeatedly using a screwdriver on the detective's computer makes infinite manuals
Related links: #6760, #6769
#### Issue bingo

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by Zth--**
## This, in theory, fixes #13719. But I used fastdmm instead of conventional ways. The big question is: is FastDMM ready for production?

_This was tested using Debian_

Computer no longer has a New() with a book attached. Instead, the book is mapped, how it always should've been.

FastDMM

### CODE REVIEW

**Review by Probe1 — APPROVED** [review]

**Comment by Zth--** [comment]
Did the same for meta and deff. Please don't merge until Probe's deff map gets merged, just in case.

**Comment by Aurx** [comment]
>Please don't merge

So what I'm seeing is that a better PR title would have been "Fastdnm test"?

**Comment by Exxion** [comment]
>All those removed duplicate varedits

Thanks, FastDMM

What the *fuck*, DM?

**Comment by Blithering** [comment]
Any chance you could replace the duplicate forensics guide on box with a space law copy?

**Comment by Zth--** [comment]
~I'll do it~ Done

**Pull request merged**
